### DISCUSSION TITLE
Rename `master` git branch to `main`

### DISCUSSION THREAD

**Discussion by blimmer** [ParabolInc/parabol#5988] [Ideas]
Many communities, both on GitHub and in the wider Git community, are renaming the default branch name of their repository from `master` -> `main` on [the recommendation of the Software Freedom Conservancy](https://sfconservancy.org/news/2020/jun/23/gitbranchname/). It would be nice to follow suit and rename the default branch of this project (and other Parabol projects as well).

GitHub now provides a UI-based way of updating the default branch, described [here](https://github.com/github/renaming#renaming-existing-branches).

The biggest things that would need to be manually addressed:

1. All local copies of the repository need to be updated to point at `main`. There's a pretty easy copy/paste solution [described in the GitHub docs](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch#updating-a-local-clone-after-a-branch-name-changes). The steps also appear as a tooltip in the GitHub UI after the rename.
2. We'd need to update any configuration that pulls the code automatically. This [code search](https://cs.github.com/?scopeName=All+repos&scope=&q=org%3AParabolInc+%22master%22) didn't result in too many results, so it seems like it might not be too impactful. cc @adaniels-parabol - would this have significant impact on infra as code?

**Comment by adaniels-parabol** [comment]
@blimmer no major issues or concerns here on infra side. There may need to be an update to public-saas side with regard to the dokku config and on some of the server, but you referenced the solution in the issue above. I support this by the way +1 for bringing this up.
